embedded/servo_bq/servo_balsaq.cc

```diff
@@ -0,0 +1,50 @@
+//%bin(servo_balsaq_main)
+#include "embedded/servo_bq/servo_balsaq.hh"
+#include "infrastructure/balsa_queue/bq_main_macro.hh"
+
+#include <chrono>
+#include <cstddef>
+#include <iostream>
+#include <sstream>
+#include <cstdlib>
+
+
+//%deps(balsa_queue)
+//%deps(message)
+
+namespace jet {
+
+ServoBq::ServoBq() {
+  set_comms_factory(std::make_unique<MqttCommsFactory>());
+}
+constexpr int PWM_FREQUENCY = 330;
```

Comment:
Put this constant in an anonymous namespace to keep it local to this file. See: embedded/servo_driver/servo_driver.cc